blocks/share-bar-block/features/share-bar/default.jsx

```diff
@@ -99,7 +83,7 @@ const ShareBarContainer = () => {
 };
 
 export const ShareBar = ({
-	customFields = {},
+	customFields,
```

Comment:
Removing the default ` = {} ` breaks the code as on line 94 the `Object.keys()` is expecting an object which in some cases `customFields` will never be an object

Reply:
@matthewroach : Thanks for confirming that! I put it back in added this test to ensure it was getting test coverage:
https://github.com/WPMedia/arc-themes-blocks/blob/TMEDIA-736-share-bar-block/blocks/share-bar-block/features/share-bar/default.test.jsx#L82-L94